src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java

```diff
@@ -834,7 +836,7 @@ public int getElementCount()
             @Override
             public void updateElement(final int index, @NotNull final Pane rowPane)
             {
-                final Permissions.Player player = users.get(index);
+                final Player player = users.get(index);
                 String rank = player.getRank().name();
                 rank = Character.toUpperCase(rank.charAt(0)) + rank.toLowerCase().substring(1);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define the locale to be used in this String operation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1449)